__tests__/lexoRank.spec.ts

```diff
@@ -64,4 +64,107 @@ describe('LexoRank', () => {
     const between = prevRank.between(nextRank);
     expect(between.toString()).toEqual(expected);
   });
+
+  it('find the nearest power', () => {
+    expect(LexoRank.nearestPowerOfTwo(1)).toEqual(1);
+    expect(LexoRank.nearestPowerOfTwo(5)).toEqual(3);
+    expect(LexoRank.nearestPowerOfTwo(6)).toEqual(3);
+    expect(LexoRank.nearestPowerOfTwo(8)).toEqual(4);
```

Comment:
Two thoughts:
* I'd like more examples - specifically, every integer from 2 to 8 - to make sure we're doing the correct thing at edge cases.
* I think `binaryDepthToInsertRanks()` would be a better name for this, since the nearest power of two for `8` is actually `3`, so the current name is potentially confusing.

Reply:
-  Added more examples for each integer between 2 to 8.
- Yes, I agree with it. I have renamed the method.